django_mesh/templates/django_mesh/media_view.html

```diff
@@ -0,0 +1,16 @@
+{% extends "django_mesh/base.html" %}
+
+{% block extra_head %}
+<link rel="alternate" type="application/json" href="{% url 'mesh_oembed' slug=media.slug %}" title="{{ media.title }}"></link>
+{% endblock %}
+
+{% block content %}
+
+<div class="media_list">
+    {% ifequal media.media_type 1 %}
```

Comment:
Instead of hardcoding this, pass in the Media class to the context and use Media.MEDIA_TYPES.NONE.

Also, make sure we're using <img> for images.